src/app/ipv/middleware.js

```diff
@@ -402,8 +411,6 @@ module.exports = {
       PAGES.NO_PHOTO_ID_SECURITY_QUESTIONS_FIND_ANOTHER_WAY,
       PAGES.PAGE_MULTIPLE_DOC_CHECK,
       PAGES.PYI_CRI_ESCAPE,
-      PAGES.PYI_ESCAPE_M2B,
-      PAGES.PYI_KBV_ESCAPE_M2B,
```

Comment:
I'm guessing this is an unrelated change - is it intentional?

Reply:
Yeah they aren't being used anymore, the pages don't exist so I think we just forgot to remove them